pkg/planner/core/core_init.go

```diff
@@ -27,10 +27,7 @@ import (
 
 func init() {
 	// For code refactor init.
-	utilfuncp.FindBestTask = findBestTask
-	utilfuncp.PruneByItems = pruneByItems
-	utilfuncp.CanPushToCopImpl = canPushToCopImpl
-	utilfuncp.PushDownTopNForBaseLogicalPlan = pushDownTopNForBaseLogicalPlan
```

Comment:
Why remove these assignments？

Reply:
after logicalop is migrated, we can call the func directly rather than a func pointer